src/GUI/ccextractorGUI.c

```diff
@@ -260,16 +269,34 @@ int main(void)
 	media.icons.img_file = icon_load("../../icon/img.png");
 	media.icons.movie_file = icon_load("../../icon/movie.png");
 #else
-    media.icons.home = icon_load("../icon/home.png");
-    media.icons.directory = icon_load("../icon/directory.png");
-    media.icons.computer = icon_load("../icon/computer.png");
-    media.icons.desktop = icon_load("../icon/desktop.png");
-    media.icons.default_file = icon_load("../icon/default.png");
-    media.icons.text_file = icon_load("../icon/text.png");
-    media.icons.music_file = icon_load("../icon/music.png");
-    media.icons.font_file =  icon_load("../icon/font.png");
-    media.icons.img_file = icon_load("../icon/img.png");
-    media.icons.movie_file = icon_load("../icon/movie.png");
+	// forming a relative path to ccextractor/linux folder
+	char path_to_ccx[PATH_MAX];
+	int ind = 0;
+	char cwd[PATH_MAX];
+	getcwd(cwd,sizeof(cwd));
```

Comment:
This gets the current dir, which might not be where CCExtractor is. What happens if CCExtractor is installed in /usr/local/bin (or maybe /usr/local/ccextractor ) and the user is just in /home/username?